### ISSUE TITLE
Direct import of stampit functions is broken with rollup.

### ISSUE THREAD

**Issue by FredyC** [stampit-org/stampit#246]
Well this is pretty nasty and got my head spinning for a bit. I am using stampit within my library which is then bundled with rollup. In the source code I have:

```js
import { init } from 'stampit';
init(...).methods(...)
```

Rollup makes this out of it:
```js
var stampit = require('stampit');
stampit.init(...).methods(...)
```

Because rollup made it like a call on object of the module instead of a direct function, the pure compose is suddenly used as it's present on `stampit.compose`. Then obviously chaining gets broken because pure compose is missing those functions.

```js
export function initializers(...args) {
  return ((this && this.compose) || stampit).call(this, {
    initializers: extractFunctions(...args)
  });
}
```

I am fairly certain that stampit shouldn't export pure compose at all. Since it's stampit export, it shouldn't provide pure compose. For anyone who wants that, just `import compose from 'stampit/compose` is way more easier to comprehend anyway.

**Comment by koresar** [comment]
Thanks! Fixed in v3.0.2

**Comment by FredyC** [comment]
I confirm it's working now. Great work, @koresar
